custom_components/zwave_mqtt/entity.py

```diff
@@ -181,4 +209,33 @@ def name(self):
     @property
     def unique_id(self):
         """Return the unique_id of the entity."""
-        return f"{self.values.primary.node.id}-{self.values.primary.value_id_key}"
+        return self.values.unique_id
+
+    @property
+    def available(self) -> bool:
+        """Return entity availability."""
+        # Use OZW Daemon status for availability.
+        instance_status = self.values.primary.ozw_instance.get_status()
+        return instance_status and instance_status.status in [
+            "driverAllNodesQueriedSomeDead",
+            "driverAllNodesQueried",
+            "driverAwakeNodesQueried",
+        ]
+
+    async def _delete_callback(self, values_unique_id):
+        """Remove this entity."""
+        if not self.values:
+            return  # race condition: delete already requested
+        if values_unique_id == self.unique_id:
+            await self.async_remove()
+
+    async def _onremove_callback(self) -> None:
```

Comment:
Since we have this here we can just name it like the entity api on remove helper method.
```suggestion
    async def async_will_remove_from_hass(self) -> None:
```